x/gov/keeper/deposit_test.go

```diff
@@ -130,3 +135,97 @@ func TestDeposits(t *testing.T) {
 		require.Len(t, deposits, 0)
 	}
 }
+
+func TestValidateInitialDeposit(t *testing.T) {
+	testcases := map[string]struct {
+		minDeposit               sdk.Coins
+		minInitialDepositPercent uint32
+		initialDeposit           sdk.Coins
+
+		expectError bool
+	}{
+		"min deposit * initial percent == initial deposit - success": {
+			minDeposit:               sdk.NewCoins(sdk.NewCoin(sdk.DefaultBondDenom, sdk.NewInt(baseDepositTestAmount))),
+			minInitialDepositPercent: baseDepositTestPercent,
+			initialDeposit:           sdk.NewCoins(sdk.NewCoin(sdk.DefaultBondDenom, sdk.NewInt(baseDepositTestAmount*baseDepositTestPercent/100))),
+		},
+		"min deposit * initial percent < initial deposit - success": {
```

Comment:
can we change all these at the end from saying "- success" to instead something like ": success", it took me a little bit to understand it wasn't a part of the equation